data/strings/types_strings.txt

```diff
@@ -11701,75 +11701,7 @@
     zh-Hans = 攀登中心
     zh-Hant = 攀登中心
 
-  [type.leisure.sports_centre.shooting]
```

Comment:
Should these translations go to type.sport.shooting?

Reply:
already exists